src/auth/constants.ts

```diff
@@ -31,6 +33,9 @@ const loginProviderTunnistusProperties: LoginProviderProps = {
   sessionPollerSettings: { pollIntervalInMs: 300000 } // 300000ms = 5min
 };
 
-export const useTunnistamoOpenIdConnect = import.meta.env.VITE_USE_TUNNISTAMO_OPENID_CONNECT === 'true' || import.meta.env.VITE_USE_TUNNISTAMO_OPENID_CONNECT === true;
+export const oidcProviderName: OidcProviderName = import.meta.env.VITE_OIDC_PROVIDER || 'tunnistus';
```

Comment:
'tunnistamo' string is assigned to an env var, but 'tunnistus' is just a plain string without a reference to a variable. Looks a bit odd, but is there a reason for it? Could it be a bit cleaner if 'tunnistus' could also be defined in variable?

Reply:
It is defined in VITE_OIDC_PROVIDER, `|| 'tunnistus'` is a fallback default value.
How do you suggest to change it?

Reply:
It could be defined as `DEFAULT_PROVIDER_NAME`. But I'm not sure, doesn't feel necessary, because when I think about it more, the string 'tunnistus' is used only here. Maybe defining it in a variable becomes necessary only if the exact 'tunnistus' string is used in several places, to avoid the pitfall of mistyping. So maybe there is no need to define it in a variable in these changes.